src/components/Signup/Signup.jsx

```diff
@@ -0,0 +1,176 @@
+  };
+
+  function checkEmail(e) {
+    let regEmail =
+      /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    regEmail.test(e.target.value) ? setEmail(true) : setEmail(false);
+  }
+
+  function checkPassword(e) {
+    const strongRegex =
+      /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#\$%\^&\*])(?=.{8,})/;
+    strongRegex.test(e.target.value) ? setPassword(true) : setPassword(false);
+    setPasswordvalue(e.target.value);
+  }
+
+  function confirmPassword(e) {
+    e.target.value === passwordValue ? setConfirm(true) : setConfirm(false);
+  }
+
+  return (
+    <div className="form">
+      <div className="form-data">
+        <h2 className="margin-b">SignUp</h2>
+        <form onSubmit={handleSignup}>
+          <div className="input input-labeled outlined margin">
+            <label className="label">First Name</label>
+            <input
+              type="text"
+              name="firstname"
+              placeholder="Enter First Name"
+              required
+            />
+          </div>
+          <div className="input input-labeled outlined margin">
+            <label className="label">Last Name</label>
+            <input
+              type="text"
+              name="lastname"
+              placeholder="Enter Last Name"
+              required
+            />
+          </div>
+          <div className="input input-labeled outlined margin">
+            <label className="label">Email Address</label>
+            <input
+              type="email"
+              name="email"
+              placeholder="user@freshpost.com"
+              onMouseLeave={checkEmail}
+              required
+            />
+          </div>
+          <div className="form-input-message margin-l">
+            {email === false && (
+              <span className="input-validation error-msg">
+                <span className="message-icon">
+                  <i className="fa fa-exclamation-circle"></i>
+                </span>
+                Please Enter Email in correct format
+              </span>
+            )}
+            {email === true && (
+              <span className="input-validation success-msg">
+                <span className="message-icon">
+                  <i className="fa fa-check-circle"></i>
+                </span>
+                Correct Email format
+              </span>
+            )}
+          </div>
+          <div className="input input-labeled outlined margin">
+            <label className="label">Password</label>
+            <input
+              type="password"
+              name="pass"
+              placeholder="******"
+              onMouseLeave={checkPassword}
+              required
+            />
+          </div>
+          <div className="form-input-message text-justify margin-l">
+            {password === false && (
+              <span className="input-validation error-msg">
```

Comment:
Make a separate component for error message

Reply:
Ok, I will check this, Thank you for review

Reply:
I will check this while adding functionality...